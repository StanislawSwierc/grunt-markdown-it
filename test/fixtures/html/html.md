# Basic Markdown with HTML

These file examples show basic markdown-it parsing without any options or syntax highlighting.

Lorem ipsum dolor sit amet, consectetur adipisicing elit. Animi dicta facilis, vitae quae nam odio ea, dolorum hic veritatis distinctio consequatur in. Quidem magni sed eum quam, maiores molestiae est debitis aliquam at atque distinctio, nam, non excepturi. Ab voluptate, ullam vero labore, quidem ex laudantium accusantium aliquam deleniti optio, quia maxime, enim dolore voluptates. Perferendis reiciendis, dignissimos, aliquam laborum labore quo autem iste rem corporis earum, eaque consequatur nemo necessitatibus nam, quia voluptas possimus harum sed doloribus iusto. Modi accusantium enim, saepe velit tempore blanditiis qui! Amet atque, quasi distinctio itaque, natus repellendus illo, obcaecati reiciendis molestiae ipsa cum.

<div class="test">
```css
.test {
    display: none;
}
```
</div>