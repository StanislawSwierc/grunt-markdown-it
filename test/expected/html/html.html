<h1>Basic Markdown with HTML</h1>
<p>These file examples show basic markdown-it parsing without any options or syntax highlighting.</p>
<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Animi dicta facilis, vitae quae nam odio ea, dolorum hic veritatis distinctio consequatur in. Quidem magni sed eum quam, maiores molestiae est debitis aliquam at atque distinctio, nam, non excepturi. Ab voluptate, ullam vero labore, quidem ex laudantium accusantium aliquam deleniti optio, quia maxime, enim dolore voluptates. Perferendis reiciendis, dignissimos, aliquam laborum labore quo autem iste rem corporis earum, eaque consequatur nemo necessitatibus nam, quia voluptas possimus harum sed doloribus iusto. Modi accusantium enim, saepe velit tempore blanditiis qui! Amet atque, quasi distinctio itaque, natus repellendus illo, obcaecati reiciendis molestiae ipsa cum.</p>
<div class="test">
<pre><code class="lang-css">.test {
    display: none;
}
</code></pre>
</div>